app/src/main/java/ch/sdp/vibester/activity/LyricsBelongGameActivity.kt

```diff
@@ -40,8 +40,9 @@ class LyricsBelongGameActivity : GameActivity() {
         if (getIntent != null) {
             gameManager = getIntent.getSerializable("gameManager") as GameManager
             // put in one line to increase coverage
-            findViewById<Button>(R.id.nextSongButton).setOnClickListener { startRound(ctx, gameManager) }
+            findViewById<Button>(R.id.nextSongLyrics).setOnClickListener { startRound(ctx, gameManager) }
```

Comment:
I'm not sure I can create a function that takes as an argument the function inside the listener, because the functions I use inside the different listeners don't all have the same arguments.